tests/suites/casper/formfill.js

```diff
@@ -73,10 +84,15 @@ casper.test.begin('fill() & fillNames() tests', 18, function(test) {
             strange:       "very"
         });
         testFormValues(test);
-        if (!skipPhantom200(test, 1)) {
+        if (!skipPhantom200(test, 2)) {
             test.assertEvalEquals(function() {
                 return __utils__.findOne('input[name="file"]').files.length === 1;
             }, true, 'can select a file to upload');
+            if (!skipSlimer095(test,1)) {
```

Comment:
Do you think it is normal that the "setFieldValue"  method doesn't work on input[type="file"] ?